packages/web-components/src/image/image.options.ts

```diff
@@ -0,0 +1,46 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * Border radius
+ * @public
+ */
+export const BorderRadius = {
```

Comment:
@halerankin should this be namespaced for Image since it is specific to image? Sorry for missing this!